webextensions/api/browserSettings.json

```diff
@@ -46,6 +46,28 @@
             }
           }
         },
+        "closeTabsByDoubleClick": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserSettings/closeTabsByDoubleClick",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "61"
+              },
+              "firefox_android": {
+                "version_added": "61"
```

Comment:
[This comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1435142#c97) suggests this should be false